release notes/v0.57.0.md

```diff
@@ -0,0 +1,180 @@
+k6 `v0.57.0` is here 🎉! This release includes:
+
+- Introducing helpers for functional testing.
+- The `k6/experimental/csv` module gets a new `asObjects` option.
+- We no longer support the `k6/experimental/browser` module.
+- Moving all non-public APIs to the `internal` package.
+- Many other improvements, bug fixes, and internal refactors.
+- (_optional_) `<highlight of breaking changes>`
+- `<Summary of new features>` (_one or multiple bullets_)
+
+
+## Breaking changes
+
+- [#4161](https://github.com/grafana/k6/pull/4161) Drops `k6/experimental/browser`. If you are still using it, please follow the [instructions](https://grafana.com/docs/k6/latest/using-k6-browser/migrating-to-k6-v0-52/) to move to the not experimental module.
```

Comment:
```suggestion
- [#4161](https://github.com/grafana/k6/pull/4161) Drops `k6/experimental/browser`. If you are still using it, please follow the [instructions](https://grafana.com/docs/k6/latest/using-k6-browser/migrating-to-k6-v0-52/) to move to the graduated and stable `k6/browser` module.
```

Reply:
@codebien suggestion is great 👍 . Two small nits, I would also replace "you are" with "you're" and remove "please".

Reply:
```suggestion
- [#4161](https://github.com/grafana/k6/pull/4161) Drops `k6/experimental/browser`. If you're still using it, follow the [instructions](https://grafana.com/docs/k6/latest/using-k6-browser/migrating-to-k6-v0-52/) to move to the graduated and stable `k6/browser` module.
```